web/src/pages/Route/components/Step3/index.tsx

```diff
@@ -32,16 +32,17 @@ type Props = {
   onChange: (data: { plugins: PluginComponent.Data; script: any, plugin_config_id?: string; }) => void;
   readonly?: boolean;
   isForceHttps: boolean;
+  isProxyEnable: boolean;
```

Comment:
could it be optional and default to false?

Reply:
yep, already fixed, thanks a lot